eks-distro-base/check_update.sh

```diff
@@ -39,15 +39,19 @@ FROM public.ecr.aws/amazonlinux/amazonlinux:$AL_TAG as builder
 
 RUN set -x && \
     if grep -q "2022" "/etc/os-release"; then \
-        yum install -y dnf-plugin-release-notification && \
-        yum check-release-update &> /tmp/new-release && \
-        if grep "A newer release" /tmp/new-release; then echo "100" > ./return_value; fi \
+        yum install -y dnf-plugin-release-notification; \
     fi
 
 RUN rm -rf /var/lib/rpm
 COPY --from=base_image /var/lib/rpm /var/lib/rpm
 COPY --from=base_image /etc/yum.repos.d /etc/yum.repos.d
 
+RUN set -x && \
+    if grep -q "2022" "/etc/os-release"; then \
```

Comment:
this has to be run after replacing the yum db otherwise it wont necessarily see a new verison.

Reply:
Should we add that as a comment?